soliket/lensing/lensing.py

```diff
@@ -3,16 +3,15 @@
 
 import numpy as np
 
-from cobaya.likelihoods._base_classes import _InstallableLikelihood
-from cobaya.conventions import _packages_path
```

Comment:
`_packages_path` is no longer set, and appears on l48 so this may fail. Is it defined elsewhere?